functions.gs

```diff
@@ -83,26 +98,33 @@ function checkCommandValidity (cmd,row,uniqueid,userid,channelid){
   var mod_userid = globvar['MOD_USERID'];
   var mention_mod = '<@'+mod_userid+'>';
 
-  var request_formatted = '<' + row.slackURL + '|request ' + row.uniqueid + '> (' + row.requesterName + ', ' + row.requesterAddr + ')';
+  var request_formatted = requestFormatted(
+    row.slackURL, uniqueid, row.requesterName, row.requesterAddr);
 
   var cmd_state_machine={ // this is the finite state machine object, which contains the return messages and codes for every possible {command,status} combination
       command:{
         "done":{
           status:{
             "Sent|FailSend|Re-open":{
               returnCode:false,
-              returnMsg:'error: You cannot complete '+request_formatted+' because it is yet to be assigned. '+
-                                           'Type `/listmine` to list the requests you are currently volunteering for in this channel. If you think there is a mistake, please contact ' + mention_mod + '.'
+              returnMsg: textToJsonBlocks(
+`error: You cannot complete ${request_formatted} because it is yet to be assigned.
+Type \`${listmine}\` to list the requests you are currently volunteering for in this channel.
```

Comment:
\`${listmine}\` should be \`/listmine\`?

Reply:
Good catch!